core/utils.py

```diff
@@ -232,6 +232,24 @@ def dict_from_yaml(yaml_str: str) -> Dict[str, Any]:
         raise InvalidInputException(e)
 
 
+def yaml_from_dict(dictionary: Dict[str, Any], width: int =80) -> str:
```

Comment:
Please add tests for this function into _utils_test.py_

Reply:
there is a test in utils_test.py  https://github.com/oppia/oppia/blob/a80d32e3854e8a7dbc8bc20a9c67768ec54c24ef/core/utils_test.py#L64-L78 , is anything more needed?

Reply:
Ah, right.